src/module.ts

```diff
@@ -186,6 +186,25 @@ export interface ModuleOptions {
     layoutFallbacks?: string[]
     injectPage?: boolean
   },
+  /**
+   * Anchor link generation config
+   *
+   * @default {}
+   */
+  anchorLinks?: boolean | {
```

Comment:
Shall we move this option under the `markdown` field?

Reply:
Sounds reasonable. We should do that.